### PR TITLE
[#503] Customize commit interval so the test environment can have a d…

### PR DESCRIPTION
**Pull request by paulodiniz**
…ifferent value

### CODE REVIEW

File: infrastructure/helm-chart/charts/apps/charts/airy-config/templates/kafka.yaml [hunk 1]
```diff
@@ -5,3 +5,4 @@ metadata:
 data:
   KAFKA_BROKERS: {{ .Values.kafka.brokers }}
   KAFKA_SCHEMA_REGISTRY_URL: {{ .Values.kafka.schemaRegistryUrl }}
+  KAFKA_COMMIT_INTERVAL_MS: {{ .Values.kafka.commitInterval }}
```
Comment (pascal-airy) [inline infrastructure/helm-chart/charts/apps/charts/airy-config/templates/kafka.yaml:8]: You could put the default here like `.Values.kafka.commitInterval | default 30000` 
so you only have it in one place

**Review by pascal-airy — APPROVED** [review]

**Review by ljupcovangelski — APPROVED** [review]
Looks good :+1:

**Review by lucapette — APPROVED** [review]
:shipit:

File: docs/docs/guides/airy-core-in-production.md [hunk 2]
```diff
@@ -66,6 +66,13 @@ We do not recommend running Kafka on docker for production environments.
 However, we provide a way to deploy the whole Kafka cluster on top of Kubernetes
 with Helm as we use this approach for test installations.
 
+
+The default commit interval is set to 1000 ms (1 second). This is *not* recommended
+for production usage.
+You change the default value to a more suitable amount on the configuration file
```
Comment (lucapette) [inline docs/docs/guides/airy-core-in-production.md:72]:
```suggestion
You change the default value to a more suitable value on the configuration file
```

File: docs/docs/guides/airy-core-in-production.md [hunk 3]
```diff
@@ -66,6 +66,13 @@ We do not recommend running Kafka on docker for production environments.
...
+`infrastructure/helm-chart/charts/apps/charts/airy-config/values.yaml` by changing
+the `commitInterval` value.
```
Comment (lucapette) [inline docs/docs/guides/airy-core-in-production.md:74]:
```suggestion
You change the `commitInterval` to a more suitable production value in the configuration file
`infrastructure/helm-chart/charts/apps/charts/airy-config/values.yaml`.
```

**Review by lucapette — DISMISSED** [review]

**Review by pascal-airy — APPROVED** [review]

**Pull request merged**